registry/storage/paths.go

```diff
@@ -242,6 +242,8 @@ func pathFor(spec pathSpec) (string, error) {
 		return path.Join(append(repoPrefix, v.name, "_uploads", v.id, "hashstates", string(v.alg), offset)...), nil
 	case repositoriesRootPathSpec:
 		return path.Join(repoPrefix...), nil
+	case referrersRootPathSpec:
+		return path.Join("/docker/registry/", "v2", "repositories", v.name, "_refs", "subjects"), nil
```

Comment:
You can use the predefined variables (like rootPrefix)

Reply:
switched to rootPrefix